docs/sources/tempo/configuration/_index.md

```diff
@@ -1222,6 +1243,100 @@ memberlist:
 
 ```
 
+## Configuration blocks
+
+Defines re-used configuration blocks.
+
+### Filter policy config
+
+Span filter config block
+
+#### Filter policy
+```yaml
+# Exclude filters (postive matching)
+[include: <policy match>]
+
+# Exclude filters (negative matching)
+[exclude: <policy match>]
+```
+
+#### Policy match
+```yaml
+# How to match the value of attributes
+# Options: "strict", "regex"
+[match_type: <string>]
+
+# List of attributes to match
+attributes: <list of policy atributes>
+
+    # Attribute key
+  - [key: <string>]
+
+    # Attribute value
+    [value: <any>]
+```
+
+#### Examples
```

Comment:
I appreciate the example, makes it much clearer :+1:

Reply:
Yeah I was decomposing it and when I looked back, even I didn't know what it was supposed to look like!

Got this specific example form the tests
https://github.com/grafana/tempo/blob/7c2e7c7e3fc7793acb7ad486bd04937d820844c2/modules/overrides/userconfigurable/api/limits_test.go#L35-L47